kf_lib_data_ingest/common/constants.py

```diff
@@ -78,53 +106,106 @@ class REFERENCE_GENOME:
         HG19 = 'hg19'
 
     class PLATFORM:
+        GENOMICS = 'Complete Genomics'
         ILLUMINA = 'Illumina'
+        ION_TORRENT = 'Ion Torrent'
+        LS454 = 'LS454'
+        PACBIO = 'PacBio'
+        SOLID = 'SOLiD'
 
```

Comment:
What are these new platforms from?

Reply:
They are from DS (https://kf-api-dataservice.kidsfirstdrc.org/#tag/SequencingExperiment%2Fpaths%2F~1sequencing-experiments%2Fpost) AND GDC (https://docs.gdc.cancer.gov/Data_Dictionary/viewer/#?view=table-definition-view&id=read_group&anchor=platform)